static/app/views/issueList/overview.tsx

```diff
@@ -940,6 +946,7 @@ class IssueListOverview extends React.Component<Props, State> {
           [query as Query]: currentQueryCount,
         },
         itemsRemoved: itemsRemoved + inInboxCount,
```

Comment:
we don't need this other stuff if we're refreshing the page

Reply:
I changed the way this works - instead of refreshing we'll just remove the issue and bring in an issue from the next page if applicable